src/ng2-auto-complete.component.ts

```diff
@@ -138,10 +140,8 @@ export class Ng2AutoCompleteComponent implements OnInit {
   /**
    * constructor
    */
-  constructor(
-    elementRef: ElementRef,
-    public autoComplete: Ng2AutoComplete
-  ) {
+  constructor(elementRef: ElementRef,
+              public autoComplete: Ng2AutoComplete) {
```

Comment:
Yea their standard is not to go over certain line count for their code.

so it's either
```
constructor(
  private alksjdghbajkhsdasd, private auoiskgdgasd
  priavte asdasda){}
```